index.bs

```diff
@@ -797,11 +795,14 @@ readable stream is <a>locked to a reader</a>.
 This abstract operation is meant to be called from other specifications that may wish to <a lt="tee a readable
 stream">tee</a> a given readable stream.
 
-The second argument, <var>cloneForBranch2</var>, governs whether or not the data from the original stream will be <a
-abstract-op lt="StructuredClone">structured cloned</a> before appearing in the second of the returned branches. This is
+The second argument, <var>cloneForBranch2</var>, governs whether or not the data from the original stream will be cloned
+(using HTML's <a>serializable objects</a> framework) before appearing in the second of the returned branches. This is
 useful for scenarios where both branches are to be consumed in such a way that they might otherwise interfere with each
-other, such as by <a abstract-op lt="Transfer">transfering</a> their <a>chunks</a>. However, it does introduce a
-noticable asymmetry between the two branches. [[!HTML]]
+other, such as by <a href="#transfer-array-buffer">transfering</a> their <a>chunks</a>. However, it does introduce a
```

Comment:
This sub-clause of the sentence is "such as", so yeah, it's just giving an example of how branches could interfere with each other.